charged-ieee/lib.typ

```diff
@@ -134,15 +134,17 @@
   })
 
   // Style bibliography.
-  show std-bibliography: set text(8pt)
+  show std-bibliography: set text(size: 8pt); set par(leading: 0.5em); set block(spacing: 0.5em)
```

Comment:
This is a bit confusing. It looks like you want the changes to all apply to the bibliography, but that's not the case. The semicolon is equivalent to them being on a separate line. 

To affect the bibliography, you'll unfortunately have to repeat the `show std-bibliography:` part.